content/docs/developer/anteaterapi/getting-started.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Getting Started
+---
+
+**This page is for Anteater API users. Potential contributors should look at the [Contributor Docs](/docs/contributor/anteaterapi).**
+
+## REST or GraphQL?
+
+The first choice you need to make when using Anteater API is between our REST API and our GraphQL API.
+The correct choice is dependent on your use case and, to some degree, on personal preference.
+It is akin to the choice of which programming language to use.
+
+## Authorization
+
+Unlike some other Web APIs, Anteater API does not require API keys or authentication, so you can start using this API immediately.
+However, there are still advantages to seeking an API key; learn how to obtain and use one [here](./keys-limits).
+
+## Choosing your stack
+
+If your application is not already fully set up to handle REST or GraphQL, you will need to add to your stack, the list of technologies powering your application.
+Examples of parts of your stack include your language (e.g. JavaScript), frameworks (e.g. React), and, most relevantly to Anteater API, your HTTP client.
+If using GraphQL, you may also want to select a specialized GraphQL client.
+
+We have a guide on integrating our REST API with TypeScript [here](/docs/developer/anteaterapi/rest-api/guides/typescript-integration).
+
+## Attribution
+
+If you use Anteater API, you **must** follow the [Attribution Policy](./docs/about/attribution-policy).
```

Comment:
Make note that the attribution policy only applies to users of the hosted API?

Reply:
IMO that's a technicality since >80% of people just want to consume our instance.

Reply:
That's a fair point since it's mentioned on that page too.